### PR TITLE
IBX-7547: Labels & screenshot corrected

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://github.com/ibexa/user/pull/71)
| Versions      | master

"Date and time" labels & screenshot corrected. 

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date

### CODE REVIEW

File: docs/getting_started/get_started.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ You can access your user settings on the right side of the top bar:
 ![User preferences menu](img/user_preferences.png)
 
 Here you can change your user password and define your user preferences,
-such as preferred timezone, date and time format, or Back Office language.
+such as preferred timezone, short of full date and time format, or Back Office language.
```
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/getting_started/get_started.md:74]:
```suggestion
such as preferred timezone, short and full date and time format, or Back Office language.
```
Both options are there to set

**Review by dabrt — APPROVED** [review]

**Review by juskora — APPROVED** [review]

File: docs/getting_started/get_started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/getting_started/get_started.md:74]: Still trying to remove an "and"
Inspired by https://learn.microsoft.com/en-us/system-center/orchestrator/standard-activities/format-date-time https://www.ibm.com/docs/fr/i/7.5?topic=items-working-date-time-data-types

```suggestion
such as preferred timezone, short and full date time format, or Back Office language.
```
or
```suggestion
such as preferred timezone, short and full date/time format, or Back Office language.
```

[the remark this answers is not in the document]
Comment (dabrt) [reply docs/getting_started/get_started.md:74]: The last one!

[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/getting_started/get_started.md:74]: Do we really need to know that there will be short and full in this early list of examples?

```suggestion
such as preferred timezone, date and time formats, or Back Office language.
```

**Pull request merged**